physutils/physio.py

```diff
@@ -542,3 +544,55 @@ def neurokit2phys(
             metadata = dict(peaks=peaks)
 
         return cls(data, fs=fs, metadata=metadata, **kwargs)
+
+
+class MRIConfig:
+    """
+    Class to hold MRI configuration information
+
+    Parameters
+    ----------
+    slice_timings : 1D array_like
+        Slice timings in seconds
+    n_scans : int
+        Number of volumes in the MRI scan
+    tr : float
+        Repetition time in seconds
+    """
+
+    def __init__(self, slice_timings=None, n_scans=None, tr=None):
+        if np.ndim(slice_timings) > 1:
+            raise ValueError("Slice timings must be a 1-dimensional array.")
+        if np.size(slice_timings) != n_scans:
```

Comment:
Here, the number of scans and the slice timings are actually two different things! The number of scans is the number of MRI volumes, but the slice timing array should reflect the order in which each slice of a single volume was acquired. This could be something like e.g. 64 slices (you can see some sample cases [here](https://www.fil.ion.ucl.ac.uk/spm/docs/tutorials/fmri/preprocessing/slice_timing/)).

Reply:
Yeap, you are right I missed this